sdk/storage/azure-storage-file/azure/storage/file/aio/directory_client_async.py

```diff
@@ -506,7 +503,7 @@ def list_handles(self, recursive=False, timeout=None, **kwargs):
             file.
         :param int max_concurrency:
             Maximum number of parallel connections to use.
-        :param int timeout:
+        :keyword int timeout:
             The timeout parameter is expressed in seconds.
```

Comment:
Need to change metadata, content_settings, validate_content, max_concurrency, and encoding to :keyword: as well.